gr-blocks/include/gnuradio/blocks/phase_shift.h

```diff
@@ -0,0 +1,46 @@
+/* -*- c++ -*- */
+/*
+ * Copyright 2020 Free Software Foundation, Inc.
+ *
+ * This file is part of GNU Radio
+ *
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ *
+ */
+
+#ifndef INCLUDED_PHASESHIFT_H
+#define INCLUDED_PHASESHIFT_H
+
+#include <gnuradio/blocks/api.h>
+#include <gnuradio/sync_block.h>
+
+namespace gr {
+namespace blocks {
+
+/*!
+ * \brief This block will shift the incoming signal by the specified radians.
```

Comment:
can you note here and above in the GRC that this block is like a multiply const, but with the const limited to abs() == 1 to provide a constant phase shift (and thus the block name)? I think that would be useful for some users. ..